thentos-core/src/Thentos/Backend/Core.hs

```diff
@@ -133,6 +133,12 @@ thentosErrorInfo other e = f e
         (Nothing, err400, "thentos session not found")
     f NoSuchServiceSession =
         (Nothing, err400, "service session not found")
+    f NoSuchPersona =
+        (Nothing, err400, "persona not found")
+    f NoSuchContext =
+        (Nothing, err400, "context not found")
+    f MultiplePersonasPerContext =
+        (Nothing, err400, "Cannot register multiple personas by a user for the same context")
```

Comment:
Will be changed in the next commit. Nevertheless, "Sybil" is a proper name, therefore the error message will still start with an uppercase letter ;)